modules/xfem/src/userobjects/MeshCut2DRankTwoTensorNucleation.C

```diff
@@ -40,18 +49,25 @@ MeshCut2DRankTwoTensorNucleation::validParams()
       "point2",
       Point(0, 1, 0),
       "End point for axis used to calculate some cylindrical material tensor quantities");
+  params.addParam<bool>(
+      "always_cut_element",
```

Comment:
I don't like this parameter name because it isn't clear what it applies to. I think it needs to have 'nucleate' in the name. Maybe something like 'always_nucleate', 'extend_nucleated_cracks_across_element', 'nucleate_across_full_element'? I think I like the last one the best.